### DISCUSSION TITLE
Is there a way to manipulate multiple files in neo-tree?

### DISCUSSION THREAD

**Discussion by nolongerhesitate** [LazyVim/LazyVim#4296] [Q&A]
Such as selecting, moving, deleting and so on.  I tried searching for the question, but I did not find any answers.

**Comment by EricDriussi** [comment]
Hi there!
Not meaning to be rude or anything but that question might be better answered in [neo-tree discussion](https://github.com/nvim-neo-tree/neo-tree.nvim/discussions).

Given [this conversation](https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/472) I would guess it's not doable at the moment, but I might be missing something :smiley:
